packages/manager/modules/carrier-sip/src/dashboard/dashboard.html

```diff
@@ -26,7 +26,7 @@
 </oui-page-header>
```

Comment:
Could you please rebase on `develop` branch?
In order to make sure that it is on top of the latest migration (see #3190)

Because `<oui-page-header>` component has been deprecated.

Thank you!

Reply:
Hello @antleblanc, will do as soon as possible

Reply:
Done